### PR TITLE
PLT-1882 Fixed issue with on-hover timestamps

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Due to the localization merges happening at the same time the on-hover timestamp was changed to use the same format as the root post time stamp.  This changes it back to using HH:MM format.  Some minor css issues remain (e.g. when zooming out very far or shrinking the screen) so I'll reassign the ticket to AM after this is in to clean up any remaining issues.

### CODE REVIEW

File: web/react/utils/utils.jsx [hunk 1]
```diff
@@ -201,40 +203,17 @@ export function displayDate(ticks) {
...
+        <FormattedTime
+            value={ticks}
+            hour='numeric'
+            minute='numeric'
+            hour12={!useMilitaryTime}
+        />
+    );
 }
 
 export function displayDateTime(ticks) {
```
Comment (coreyhulen) [inline web/react/utils/utils.jsx]: +1

this is used on the system console?  can it handle a <FormattedTime /> return?

Comment (rgarmsen2295) [inline web/react/utils/utils.jsx]: I was really confused for a bit as I grep'd my local copy again looking for other uses of `Utils.displayTime` and couldn't find any (there weren't any when I originally made my changes), but it appears @jwilander's PR that added it's use was merged an hour after I submitted this PR.  Good catch, thanks @coreyhulen, I'll change this tomorrow when I'm in.

**Comment by rgarmsen2295** [comment]
Okay changed it so a pre-formatted and a string version exist, please re-review when possible @coreyhulen @jwilander, thanks!

**Comment by jwilander** [comment]
+1

**Pull request merged**